### PR TITLE
[review & merge] fix: response id and getTrail rename

### PR DESCRIPTION
**Pull request by McCloudFace**
- [x] fix data response. `trail_id` now `id` as a string
- [x] change `getAtrail` to just `getTrail` cause naming things is hard

@cjm771 or @weber93 Review & merge?  plz & ty!

### CODE REVIEW

File: database/index.js [hunk 1]
```diff
@@ -14,15 +14,15 @@ var knex = require('knex')({
 // result row to the callback
 // called from ../server/index.js
 
-var getAtrail = (id, callback) => {
+var getTrail = (id, callback) => {
```
Comment (cjm771) [inline database/index.js:17]: image
gonna wanna revise ur tests

**Review by cjm771 — CHANGES_REQUESTED** [review]
looks straight foward, revise your tests and likely all should be good.

File: database/index.js (hunk 1 above)
Comment (McCloudFace) [reply database/index.js:17]: @cjm771 Thanks, nice catch. Fixed!

**Comment by cjm771** [comment]
cool! 😇

**Pull request merged**